From a14162d8da62fb570df916d7386febe51d6ed2bc Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.org>
Date: Wed, 31 Jul 2019 17:41:47 +0100
Subject: [PATCH] overlays: Update the upstream overlay

The recent vc4-kms-v3d commit has changed the content of the
upstream overlay (even though the extra fragment is disabled).

Signed-off-by: Phil Elwell <phil@raspberrypi.org>
---
 arch/arm/boot/dts/overlays/upstream-overlay.dts | 6 ++++++
 1 file changed, 6 insertions(+)

--- a/arch/arm/boot/dts/overlays/upstream-overlay.dts
+++ b/arch/arm/boot/dts/overlays/upstream-overlay.dts
@@ -110,6 +110,12 @@
 		};
 	};
 	fragment@17 {
+		target = <&hdmi>;
+		__dormant__ {
+			dmas;
+		};
+	};
+	fragment@18 {
 		target = <&usb>;
 		#address-cells = <1>;
 		#size-cells = <1>;
